server/.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+dist
```

Comment:
Keeping separate .dockerignore files works better since client and server have different contexts and needs. A global file would also require changes to the Dockerfiles.